spring/opengemini-spring/src/main/java/io/opengemini/client/spring/data/annotations/RetentionPolicy.java

```diff
@@ -0,0 +1,29 @@
+package io.opengemini.client.spring.data.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Retention(java.lang.annotation.RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface RetentionPolicy {
```

Comment:
This class name is identical to other class names and can only be distinguished by the package name, which makes it difficult to use. Such as `io/opengemini/client/api/RetentionPolicy.java`